unittest/test_Util.cpp

```diff
@@ -590,7 +590,7 @@ TEST_CASE("Util::make_relative_path")
   SUBCASE("Path matches neither actual nor apparent CWD")
   {
 #ifdef _WIN32
-    CHECK(make_relative_path("C:/", "C:/a", "C:/b", "C:/x") == "C:/x");
+    CHECK(make_relative_path("C:/", "C:/a", "C:/b", "C:/x") == "../x");
```

Comment:
Hm, this doesn't look right. The (I guess badly named) test case is about verifying what happens when neither the actual nor the apparent current working directory _exists_ (so in the non-Windows case, ccache cannot prove that `../x` would end up in `/x` from `/a` or `/b`).

Since the whole thing about apparent CWD isn't relevant on Windows, I think we could just as well remove the `_WIN32` version of the test case if it doesn't make sense. (Note: The actual logic of rewriting paths when CWD does exist is verified in the `Util::get_relative_path` tests.)